### PR TITLE
feat(core): always use child webviews on the unstable feature

### PR DESCRIPTION
**Pull request by lucasfernog**
This PR changes the WebviewWindow implementation to always use wry's child webview implementation when the `unstable` feature flag is enabled. This enhances flexibility for app developers, allowing a WebviewWindow to be transformed to a multiwebview window and vice versa, reparent a webview from a WebviewWindow to another window etc.

### CODE REVIEW

File: core/tauri-runtime-wry/src/lib.rs [hunk 1]
```diff
@@ -3427,6 +3449,9 @@ fn create_window<T: UserEvent, F: Fn(RawWindow) + Send + 'static>(
 
   if let Some(webview) = webview {
     webviews.push(create_webview(
+      #[cfg(feature = "unstable")]
+      WebviewKind::WindowChild,
```
Comment (amrbashir) [inline core/tauri-runtime-wry/src/lib.rs:3453]: Why do we want to use `WindowChild` for `WebviewWindow` ? I think we still has issues with that on macOS

Comment (lucasfernog) [reply core/tauri-runtime-wry/src/lib.rs:3453]: it's explained in the PR description, note this is all behind the unstable feature flag

Comment (amrbashir) [reply core/tauri-runtime-wry/src/lib.rs:3453]: yeah but in the future we plan to remove this `unstable` feature, no?

Comment (lucasfernog) [reply core/tauri-runtime-wry/src/lib.rs:3453]: yeah when we sort all the bugs and stabilize the API (that's why i mentioned the flag)

Comment (amrbashir) [reply core/tauri-runtime-wry/src/lib.rs:3453]: cool

**Review by amrbashir — APPROVED** [review]

**Pull request merged**